build.gradle

```diff
@@ -63,7 +65,7 @@ ext {
     archLifecycleExtentions = "androidx.lifecycle:lifecycle-extensions:$archLifecycle"
     archLifecycleCompiler = "androidx.lifecycle:lifecycle-compiler:$archLifecycle"
```

Comment:
The original benchmark used an older version of archLifecycle (`2.0.0` vs our `2.2.0-alpha02`). Is that a problem?

Reply:
Dagger should be upgraded in order to support incremental kapt (not sure why we use older one). Using `archLifecycle` 2.2.0-alpha02 also sounds good.

Reply:
I've just checked, and for the Kotlin-version of Santa-tracker we do not run incremental benchmarks, so that's why this does not matter for us. If you'd like to measure incremental scenarios, please do update dagger.